README.md

```diff
@@ -364,9 +364,22 @@ Code changes can readily be verified by running the growing collection of includ
 
 Wasabi can readily be packaged as installable *rpm* or *deb* distributions and deployed at scale as follows:
 
+> Package by running integration tests 1st:
+
+```bash
+% ./bin/wasabi.sh start package
```

Comment:
@iizrailevsky Is 'start' needed while packaging (./bin/wasabi.sh start package) ? 
If yes then, does it needed while packing without integration test ?